config.js

```diff
@@ -13,7 +13,12 @@ module.exports = require('rc')(
       verificationUrl: 'https://accounts.firefox.com/v1/verify_email',
       passwordResetUrl: 'https://accounts.firefox.com/v1/complete_reset_password',
       accountUnlockUrl: 'https://accounts.firefox.com/v1/complete_unlock_account',
-      initiatePasswordResetUrl: 'https://accounts.firefox.com/v1/reset_password'
+      initiatePasswordResetUrl: 'https://accounts.firefox.com/reset_password',
+      initiatePasswordChangeUrl: 'https://accounts.firefox.com/settings/change_password',
+      syncUrl: 'https://www.mozilla.org/en-US/firefox/sync/',
+      androidUrl: 'https://www.mozilla.org/en-US/firefox/android/',
+      iosUrl: 'https://www.mozilla.org/en-US/firefox/ios/',
```

Comment:
I wonder if we should name these with a little more detail about what they're for, e.g. "syncProductPageUrl".  Maybe I just have a thing about names..